<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-0T3LLVM" failures="1" tests="3" name="Smoke Test" time="11.325" errors="0" timestamp="2023-02-23T21:38:42 EST">
  <testcase classname="com.exelenter.testcases.LoginTest" name="validAdminLogin" time="1.303"/>
  <testcase classname="com.exelenter.testcases.LoginTest" name="validUserAnEmptyPassword" time="0.26">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.exelenter.testcases.LoginTest.validUserAnEmptyPassword(LoginTest.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- validUserAnEmptyPassword -->
  <testcase classname="com.exelenter.testcases.LoginTest" name="validUserInvalidPassword" time="0.0">
    <skipped/>
  </testcase> <!-- validUserInvalidPassword -->
  <testcase classname="com.exelenter.testcases.AddEmployeeFromExcel" name="addEmployeeTest" time="0.0">
    <ignored/>
  </testcase> <!-- addEmployeeTest -->
  <testcase classname="com.exelenter.testcases.AddEmployeeTest" name="addEmployeeTest" time="0.0">
    <ignored/>
  </testcase> <!-- addEmployeeTest -->
</testsuite> <!-- Smoke Test -->
